opm/autodiff/FlowMainEbos.hpp

```diff
@@ -23,6 +23,12 @@
 #ifndef OPM_FLOW_MAIN_EBOS_HEADER_INCLUDED
 #define OPM_FLOW_MAIN_EBOS_HEADER_INCLUDED
 
+#include <sys/utsname.h>
+#include <unistd.h>
+#include <ctime>
+#include <sys/statvfs.h>
+#include <sys/sysinfo.h>
```

Comment:
Unfortunately this header seems to be a linux-specific thing. It is only used to extract the RAM size of the system. The simplest solution is to simply drop that particular output, but if you think it is very important I can try to find a cross-platform solution.

So: is it necessary to have RAM output?